src/models/connectionProfile.ts

```diff
@@ -53,6 +53,7 @@ export class ConnectionProfile extends ConnectionCredentials implements IConnect
         connectionStore: ConnectionStore,
         context: vscode.ExtensionContext,
         accountStore?: AccountStore,
+        azureController?: AzureController,
```

Comment:
This should never be undefined I would think (just put it before the azureController to avoid the compile error)